cpu/cc2538/vectors.c

```diff
@@ -71,74 +71,74 @@ WEAK_DEFAULT void isr_dmaerr(void);
 /* interrupt vector table */
 ISR_VECTORS const void *interrupt_vector[] = {
     /* Exception stack pointer */
-    (void*) (&_estack),             /* pointer to the top of the stack */
+    __extension__ (void*) (&_estack),             /* pointer to the top of the stack */
```

Comment:
But I'm not completely happy with this solution either. That's why I put it into a separate commit. I can take it out of this PR, if you want.